README.md

```diff
@@ -163,7 +178,9 @@ Look at <code>core/settings/project.py</code> and make sure you update or overri
 
 **GARMIN_CONFIG** = 'absolute path to utils/conf/garmin_config.xml'
 
-**OVERPASS_API_URL** = 'url of your local overpass api endpoint (see Overpass API below)'
+**OVERPASS_API_URL** = 'http://overpass-api.de/api/interpreter'
```

Comment:
Please remove this, you can probably use public overpass api server for 'testing', and it could be added to the documentation, but not like this